presets/4.5/leds/LED VTX  race mode.txt

```diff
@@ -0,0 +1,74 @@
+#$ TITLE: RACE mode LED settings
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: LEDS
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: leds, race, vtx, colour change, sugarK, spec, spec racing, tinytrainer
+#$ AUTHOR: sugarK
+
+#$ DESCRIPTION: Sets LED settings to change colours when VTX channel is changed.
+#$ DESCRIPTION: This preset is required with HDzero VTX firmware 1.30 and above. See BF pr here https://github.com/betaflight/betaflight/pull/13096#discussion_r1411284423
+#$ DESCRIPTION:
+#$ WARNING: This preset will not work with HDzero firmware 1.20 or older
```

Comment:
Just a minor note.
This will work for older hdzero firmwares as well... Just using BF's internal variables to control LEDs.

Reply:
ill delete the warning then thanks